src/main/java/com/coreos/jetcd/resolver/SimpleEtcdNameResolverFactory.java

```diff
@@ -46,4 +49,24 @@ public String getDefaultScheme() {
   public String name() {
     return NAME;
   }
+
+  public static SimpleEtcdNameResolverFactory forEndpoints(String... endpoints) {
+    return forEndpoints(Arrays.asList(endpoints));
+  }
+
+  public static SimpleEtcdNameResolverFactory forEndpoints(List<String> endpoints) {
```

Comment:
prefer not to have `Etcd` keyword in naming unless you think that it's more clear.